scanomatic/ui_server_data/js/ccc/containers/CCCEditorContainer.jsx

```diff
@@ -25,8 +25,9 @@ export default class CCCEditorContainer extends React.Component {
   }
 
   handleFinishUpload(newImage) {
-    const plates = [...this.state.plates];
-    for (let i = 1; i <= this.state.platesPerImage; i += 1) {
+    const prevState = this.state;
```

Comment:
it should be considered the current state here and not previous state

Reply:
Changed the name, but only in this instance. Let me know if you think it is badly named elsewhere as well.